app/src/main/java/com/babylon/wallet/android/presentation/navigation/NavigationHost.kt

```diff
@@ -515,5 +520,16 @@ fun NavigationHost(
                 navController.popBackStack(MAIN_ROUTE, false)
             }
         )
+        relinkConnectors(
+            onContinueClick = {
+                navController.popBackStack()
+                navController.linkedConnectorsScreen(
+                    shouldShowAddLinkConnectorScreen = true
+                )
```

Comment:
Does this always work @sergiupuhalschi-rdx ? Should it be better to navigate to `linkedConnectorsScreen` by removing current screen from stack in the same transaction? I am not sure that all devices will be able to keep up with this.

Reply:
Good point. It was lazy of me to leave it like this :)

Reply:
Btw, from my experience, this always worked. But it's worth changing anyway!